projects/storefrontlib/cms-components/product/product-list/product-facet-navigation/active-facets/active-facets.component.html

```diff
@@ -1,9 +1,9 @@
 <ng-container *ngIf="facetList$ | async as facetList">
   <!-- TODO:(CXSPA-1695) for next major release update below feature flags -->
   <ng-container *cxFeatureLevel="'5.2'">
-    <h2 *ngIf="facetList.activeFacets && facetList.activeFacets.length > 0">
+    <div id="groupName" *ngIf="facetList.activeFacets && facetList.activeFacets.length > 0">
```

Comment:
`groupName` does not seem to be a sufficient unique ID string...

Reply:
yap... i will change to `cx-active-facets-groupName`

Reply:
"groupName" is what came with the example suggestion, but please follow the standard naming conventions for IDs. Ask a senior angular dev if in doubt.